speechbrain/pretrained/interfaces.py

```diff
@@ -219,18 +219,15 @@ def _prepare_modules(self, freeze_params):
             for p in self.mods.parameters():
                 p.requires_grad = False
 
-    def load_audio(self, path, savedir="."):
+    def load_audio(self, path):
         """Load an audio file with this model"s input spec
 
         When using a speech model, it is important to use the same type of data,
         as was used to train the model. This means for example using the same
         sampling rate and number of channels. It is, however, possible to
         convert a file from a higher sampling rate to a lower one (downsampling).
         Similarly, it is simple to downmix a stereo file to mono.
-        The path can be a local path, a web url, or a link to a huggingface repo.
         """
-        source, fl = split_path(path)
-        path = fetch(fl, source=source, savedir=savedir)
```

Comment:
removing it will avoid this problem but also deactivate other features ;-)
the bug fix needs to happen elsewhere, as this fix will break other things (e.g. resolving HuggingFace URLs)